includes/shortcodes/class-sensei-legacy-shortcodes.php

```diff
@@ -16,6 +14,8 @@
  */
 class Sensei_Legacy_Shortcodes {
 
+	const DOCS_SHORTCODE_URL = 'https://senseilms.com/documentation/';
```

Comment:
Let's take them right to the shortcodes documentation at https://senseilms.com/documentation/shortcodes/.

Reply:
I originally had that but then thought the widgets would replace some of these and wasn't sure, but I can change it back.

Reply:
Well, I guess we can get them halfway there then 🙂. I just don't want users to have to hunt for the shortcodes documentation, which they will have to do if we drop them on the landing page.

Reply:
Changed in 1e865f3c05b24b6526e09e11da089c997f572553